pennylane_lightning/core/src/lightning.kokkos.toml

```diff
@@ -0,0 +1,79 @@
+schema = 1
+
+[device]
+name = "lightning.kokkos"
+version = "v0.31.0"
+precision = ['float32', 'float64']
+
+[operations]
+observables = [
+	"NamedObs",
+	"HermitianObs",
+	"TensorObs",
+	"HamiltonianObs",
+]
+
+[[operations.gates]]
+full = [
+	"Identity",
+	"PauliX",
+	"PauliY",
+	"PauliZ",
+	"Hadamard",
+	"S",
+	"T",
+	"PhaseShift",
+	"RX",
+	"RY",
+	"RZ",
+	"Rot",
+	"CNOT",
+	"CY",
+	"CZ",
+	"SWAP",
+	"IsingXX",
+	"IsingXY",
+	"IsingYY",
+	"IsingZZ",
+	"ControlledPhaseShift",
+	"CRX",
+	"CRY",
+	"CRZ",
+	"CRot",
+	"Toffoli",
+	"CSWAP",
+	"MultiRZ",
+]
+# QubitUnitary should never be here.
+# Gates in "matrix" are decomposed to QubitUnitary.
+# Having a QubitUnitary here would mean an infinite loop.
+matrix = [
+	"OrbitalRotation",
+	"SingleExcitation",
+	"SingleExcitationMinus",
+	"SingleExcitationPlus",
+	"MultiControlledX",
+	"DoubleExcitation",
+	"DoubleExcitationMinus",
+	"DoubleExcitationPlus",
+	# TODO:
+	# Why is qml.ops.Controlled not in qml namespace?
```

Comment:
This should become supported after [#516](https://github.com/PennyLaneAI/pennylane-lightning/pull/516) is merged, albeit with a different interface.